<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpWindow</class>
 <widget class="QMainWindow" name="HelpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Follow &lt;span style=&quot; font-family:'Arial','sans-serif'; font-weight:700;&quot;&gt;File Conversion – Manual Annotation – Training Data Creation&lt;/span&gt;&lt;span style=&quot; font-weight:700;&quot;&gt; &lt;/span&gt;in &lt;span style=&quot; font-style:italic;&quot;&gt;Manual Seafloor Mapping&lt;/span&gt; section for training data creation.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Follow &lt;span style=&quot; font-family:'Arial','sans-serif'; font-weight:700;&quot;&gt;File Conversion – Manual Annotation – Output Data&lt;/span&gt; in &lt;span style=&quot; font-style:italic;&quot;&gt;Manual Seafloor Mapping&lt;/span&gt; section for &lt;span style=&quot; font-family:'Arial','sans-serif';&quot;&gt;annotating seafloor data without utilizing our model.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Arial','sans-serif';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Arial','sans-serif';&quot;&gt;Follow &lt;/span&gt;&lt;span style=&quot; font-family:'Arial','sans-serif'; font-weight:700;&quot;&gt;Data Preprocessing – Model Prediction – Manual Annotation – Output Data&lt;/span&gt;&lt;span style=&quot; font-family:'Arial','sans-serif';&quot;&gt; in &lt;/span&gt;&lt;span style=&quot; font-family:'Arial','sans-serif'; font-style:italic;&quot;&gt;Auto Seafloor Mapping&lt;/span&gt;&lt;span style=&quot; font-family:'Arial','sans-serif';&quot;&gt; section for automatic seaflloor mapping using our model.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>375</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
